config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "guest/:id", to: "guests#show"
+  post "guest/:id/add_room", to: "guests#add_room"
```

Comment:
These routes are RESTful overall, good job - I would change this route to reflect the comment I made above about where we should be creating a new instance of Booking. I would do something like: 

```
post "/guests/:id/rooms", to: "bookings#create"
```

Also, remember that the resources in our routes should always be plural; so `guests` just like you have `rooms` on line 10.